assets/js/googlesitekit-dashboard-details.js

```diff
@@ -35,14 +35,12 @@ import { Component, render } from '@wordpress/element';
 import { loadTranslations } from './util';
 import './components/notifications';
 import DashboardDetailsApp from './components/dashboard-details/dashboard-details-app';
-import ErrorHandler from './components/ErrorHandler';
+import Root from './components/root';
 
 class GoogleSitekitDashboardDetails extends Component {
 	render() {
 		return (
-			<ErrorHandler>
-				<DashboardDetailsApp />
-			</ErrorHandler>
+			<DashboardDetailsApp />
```

Comment:
This can be inlined below as `GoogleSitekitDashboardDetails` is an unnecessary wrapper now.